packages/replay-web/package-lock.json

```diff
@@ -608,6 +608,12 @@
 				"chalk": "^3.0.0"
 			}
 		},
+		"@replay/core": {
```

Comment:
Let's not commit this package here (it should be linked with lerna, see [CONTRIBUTING](https://github.com/edbentley/replay/blob/master/CONTRIBUTING.md)).